packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/Providers/ModelProvider.cs

```diff
@@ -624,7 +697,7 @@ private IEnumerable<FieldProvider> GetAllBaseFieldsForConstructorInitialization(
         }
 
         private (IReadOnlyList<ParameterProvider> Parameters, ConstructorInitializer? Initializer) BuildConstructorParameters(
-            bool isPrimaryConstructor)
+            bool isPrimaryConstructor, bool includeDiscriminatorParameter = false)
```

Comment:
wonder if we should rename this param from isPrimaryConstructor to isInitializationConstructor.